ocamltest/environments.ml

```diff
@@ -62,6 +62,15 @@ let lookup_as_bool variable env =
   | Some "true" -> Some true
   | Some _ -> Some false
 
+let lookup_as_int variable env =
+  match lookup variable env with
+  | None -> None
+  | Some value ->
+      begin match int_of_string value with
+      | exception _ -> None
+      | n -> Some n
+      end
```

Comment:
Since 4.05 we have `int_of_string_opt` in the standard library...

Reply:
I amended my patch (still in Xavier's branch) to use `int_of_string_opt` as suggested, but not `Option.bind`. With the explicit pattern-match the code looks similar to the other functions around and, for example, it is clear that the `| None -> ...` case corresponds to what happens if the variable is not found. The `Option.bind` rewrite is more compact but makes code reasoning less obvious.